init.go

```diff
@@ -1,12 +1,23 @@
 package main
 
 /*
-usage: fn init <name>
-
-If there's a Dockerfile found, this will generate the basic file with just the image name. exit
-It will then try to decipher the runtime based on the files in the current directory, if it can't figure it out, it will ask.
-It will then take a best guess for what the entrypoint will be based on the language, it it can't guess, it will ask.
-
+ usage: fn init <name>
+
+ o If there's a Dockerfile found, this will force the 'runtime'
+   to be 'docker' and a basic function file with the image name
+   and mininum fields like following will be generated, for example:
+
+   name: dockeruser/hello
```

Comment:
We're not putting the `dockeruser` here anymore, as of last week, see: https://github.com/fnproject/fn/pull/237

Also, see main quickstart.

Reply:
updated comment to have 'name: hello' instead of 'name: dockeruser/hello'